internal/services/validation_test.go

```diff
@@ -0,0 +1,243 @@
+package services
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/go-playground/validator/v10"
+)
+
+type TestInThePastStruct struct {
+	Date string `validate:"InThePast"`
+}
+type TestCommonPasswordStruct struct {
+	CommonPassword string `validate:"hasCommonWord"`
+}
+type TestSequenceStruct struct {
+	SequencePassword string `validate:"hasSequence"`
+}
+type TestDigitStruct struct {
+	DigitPassword string `validate:"hasDigit"`
+}
+type TestSpecialCharStruct struct {
+	SpecialCharPassword string `validate:"hasSpecialChar"`
+}
+
```

Comment:
you don't really need those, you can do it the same way as [here](https://github.com/mo-pm888/twitter/pull/58/files#diff-b7856ed2022e3f4fbf3c76500aac54e3cdb1f107c522708d27fa84eefc705a20R218)